tests/foreman/cli/test_leapp.py

```diff
@@ -0,0 +1,210 @@
+"""Test for LEAPP cli
+
+:Requirement: leapp
+
+:CaseLevel: Acceptance
+
+:CaseComponent: LEAPP
+
+:Team: Rocket
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:CaseAutomation: Automated
+
+:Upstream: No
+"""
+import pytest
+from wrapanapi.entities.vm import VmState
+
+from robottelo.constants import PRDS
+
+CUSTOM_REPOSET = {
+    'rhel8_8_bos': 'Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)',
+    'rhel8_8_aps': 'Red Hat Enterprise Linux 8 for x86_64 - AppStream (RPMs)',
+    'rhel9_2_bos': 'Red Hat Enterprise Linux 9 for x86_64 - BaseOS (RPMs)',
+    'rhel9_2_aps': 'Red Hat Enterprise Linux 9 for x86_64 - AppStream (RPMs)',
+}
+CUSTOM_REPOS = {
+    'rhel8_8_bos': {
+        'id': 'rhel-8-for-x86_64-baseos-rpms',
+        'name': 'Red Hat Enterprise Linux 8 for x86_64 - BaseOS RPMs 8.8',
+        'releasever': '8.8',
+        'version': '8',
+        'reposet': CUSTOM_REPOSET['rhel8_8_bos'],
+        'product': PRDS['rhel8'],
+        'distro': 'rhel8',
+        'key': 'rhel8_bos',
+        'basearch': 'x86_64',
+    },
+    'rhel8_8_aps': {
+        'id': 'rhel-8-for-x86_64-appstream-rpms',
+        'name': 'Red Hat Enterprise Linux 8 for x86_64 - AppStream RPMs 8.8',
+        'releasever': '8.8',
+        'basearch': 'x86_64',
+        'version': '8',
+        'reposet': CUSTOM_REPOSET['rhel8_8_aps'],
+        'product': PRDS['rhel8'],
+        'distro': 'rhel8',
+        'key': 'rhel8_aps',
+    },
+    'rhel9_2_bos': {
+        'id': 'rhel-9-for-x86_64-baseos-rpms',
+        'name': 'Red Hat Enterprise Linux 9 for x86_64 - BaseOS RPMs 9.2',
+        'releasever': '9.2',
+        'version': '9',
+        'reposet': CUSTOM_REPOSET['rhel9_2_bos'],
+        'product': PRDS['rhel9'],
+        'distro': 'rhel9',
+        'key': 'rhel9_bos',
+        'basearch': 'x86_64',
+    },
+    'rhel9_2_aps': {
+        'id': 'rhel-9-for-x86_64-appstream-rpms',
+        'name': 'Red Hat Enterprise Linux 9 for x86_64 - AppStream RPMs 9.2',
+        'releasever': '9.2',
+        'basearch': 'x86_64',
+        'version': '9',
+        'reposet': CUSTOM_REPOSET['rhel9_2_aps'],
+        'product': PRDS['rhel9'],
+        'distro': 'rhel9',
+        'key': 'rhel9_aps',
+    },
+}
```

Comment:
Inside constants.py, REPOSET, REPOS are defined only for major version (for example rhel7, rhel8, rhel9, etc) with that release verion only 7.0, 8.0, & 9.0 repos would get enable.
So for minor versions like 7.9, 8.8, 9.2 I needed to re-defined these and these constants are only be using in Leapp sceanrios. So keeping in the same test module as per suggestion from other rocket team member.

Reply:
@vijaysawant I think content repos always grab the latest rhel version so with release versions 7, 8, and 9, It will grab 7.9, 8.8, 9.2 versions respectively.

Reply:
- Talking about the latest versions (i.e 7.9, 8.8 & 9.2 ) - I was also expecting that the latest repos would get enabled and synch if we use robottelo > constants like REPOSET['rhel8_bos'], REPOSET['rhel8_aps'], etc.
- But only repository for major versions got enabled, which ended with reposet error, hence to be more specific with major.minor version we are enabling specific repos explicitly.